Classes/FormElements/Recaptcha.php

```diff
@@ -81,24 +81,71 @@ public function onSubmit(FormRuntime $formRuntime, &$elementValue)
         }
 
         $properties = $this->getProperties();
-        $recaptcha = new \ReCaptcha\ReCaptcha($properties['secretKey'], $requestMethod);
+
+        /**
+         * If Action and Threshold are set it is the V3 Captcha.
+         * The Request Method is not used by the V3 Captcha.
+         */
+        if(isset($properties['action']) && isset($properties['threshold'])) {
```

Comment:
Are you really sure, that the request method is not used any more by the V3 Captcha? This is an option of the PHP Implementation on how to connect to the endpoint and nothing documented in the Documentations.

Reply:
No, I'm not really sure, it could be that it's still supported. I didn't found any documentation about this, so I might just be wrong at that Point.

Reply:
Should be still supported as this is only the method used by the server to connect to the google api. See [google/recaptcha](https://github.com/google/recaptcha/blob/master/src/ReCaptcha/ReCaptcha.php#L125).

Reply:
Made this a thing again!